ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.js

```diff
@@ -93,7 +93,7 @@ export default function ConfirmationNetworkSwitch({ newNetwork }) {
             size={64}
           />
           <Typography
-            color={COLORS.BLACK}
+            color={COLORS.TEXT_DEFAULT}
             variant={TYPOGRAPHY.H6}
             fontWeight={FONT_WEIGHT.NORMAL}
             align={TEXT_ALIGN.CENTER}
```

Comment:
Use font-awesome icons in favor of svgs to ensure we have the correct licensing and allows us to access the css to apply the theme compatible colors via css